cmd/lifecycle/creator.go

```diff
@@ -154,12 +161,16 @@ func (c *createCmd) Exec() error {
 	if api.MustParse(c.platform.API()).Compare(api.MustParse("0.7")) >= 0 {
 		cmd.DefaultLogger.Phase("ANALYZING")
 		analyzedMD, err = analyzeArgs{
-			imageName: c.previousImage,
-			keychain:  c.keychain,
-			layersDir: c.layersDir,
-			platform:  c.platform,
-			useDaemon: c.useDaemon,
-			docker:    c.docker,
+			additionalTags:   c.additionalTags,
+			cacheImageRef:    c.cacheImageRef,
+			docker:           c.docker,
+			keychain:         c.keychain,
+			layersDir:        c.layersDir,
+			outputImageRef:   c.outputImageRef,
```

Comment:
This might not be needed... it is only used to validate registry access and I think that happens before `analyze()` is called. But at the same time, it doesn't hurt to have it and it might save us from creating bugs later.

Reply:
See https://github.com/buildpacks/lifecycle/pull/646